torch/_dynamo/variables/functions.py

```diff
@@ -693,10 +694,18 @@ def call_function(
         # call_function must check any unsupported arguments and graph-break.
         # It's safe to assume args/kwargs from orig_fn map 1:1 to args/kwargs of remapped_fn,
         # since that's the contract for putting a mapping in `traceable_collective_remaps`
+        import torch.distributed as dist
+
         if "async_op" in kwargs and kwargs["async_op"].as_python_constant():
             unimplemented(
                 f"CollectiveFunctionRewriteVariable can't support async_op=True for {self.fn}"
             )
+        group = kwargs.get("group")
+        if group is None or (
+            isinstance(group, variables.ConstantVariable)
+            and group.as_python_constant() is None
+        ):
+            kwargs["group"] = ProcessGroupVariable(dist.group.WORLD, source=self.source)
```

Comment:
NOTE: this is wrong. We need to popluate the source and install the guard properly.

Reply:
Yea I guess, world shouldn't change but it's proper to guard.